llarp/crypto/crypto.hpp

```diff
@@ -22,16 +22,16 @@ namespace llarp
     maybe_decrypt_name(std::string_view ciphertext, SymmNonce nonce, std::string_view name);
 
     /// xchacha symmetric cipher
-    bool
-    xchacha20(uint8_t*, size_t size, const SharedSecret&, const TunnelNonce&);
-    bool
-    xchacha20(uint8_t*, size_t size, const uint8_t*, const uint8_t*);
+    static bool
+    xchacha20(uint8_t* buf, size_t buf_size, const SharedSecret&, const TunnelNonce&);
+    static bool
```

Comment:
This change to `static` shouldn't be here (it was frankensteined from the other commit that changed to a namespace).